src/main/java/com/movie_collection/bll/services/CategoryService.java

```diff
@@ -1,6 +1,37 @@
 package com.movie_collection.bll.services;
 
+import com.movie_collection.be.Category;
 import com.movie_collection.bll.services.interfaces.ICategoryService;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 public class CategoryService implements ICategoryService {
+
+    Category[] categories = {
+            new Category(1, new SimpleStringProperty("Action")),
+            new Category(2, new SimpleStringProperty("Horror"))
+    };
+    List<Category> list = new ArrayList<>(Arrays.asList(categories));
```

Comment:
why not instantiate a new ArrayList straightaway and just use .add?

Reply:
Its just mock data it will be deleted cuz we will have proper data from db

Reply:
Also I did it like this because it was not possible to throw it in the default constructor we have because or the configuration currently 
But aa I sad it was just for testing I will get rid of it I just wanted u guys to see the logic behind it